src/v2i-hub/SpatPlugin/test/TestNTCIP1202.cpp

```diff
@@ -32,52 +36,38 @@ TEST(NTCIP1202Test, copyBytesIntoNtcip1202)
     ntcip1202_p->copyBytesIntoNtcip1202(buf, numBytes);
 
     SPAT *spat_ptr = (SPAT *)calloc(1, sizeof(SPAT));
-    ntcip1202_p->ToJ2735r41SPAT(spat_ptr, reinterpret_cast<char *>(update_to_intersection_name->buf), update_to_intersection_id->id);
+    ntcip1202_p->ToJ2735SPAT(spat_ptr,tsMsec, "test intersection name", 9012);
+
     ASSERT_EQ(3,  spat_ptr->intersections.list.array[0]->states.list.array[0]->state_time_speed.list.array[0]->eventState);
+
     free(spat_ptr);
 }
 
-TEST(NTCIP1202Test, ToJ2735r41SPAT)
+TEST(NTCIP1202Test, ToJ2735SPAT)
 {
-    auto clock = std::make_shared<CarmaClock>();
-    clock->wait_for_initialization();
-    auto ntcip1202_p = std::make_shared<Ntcip1202>(clock);
-    SPAT *spat_ptr = (SPAT *)calloc(1, sizeof(SPAT));
+    uint64_t tsMsec = 1677775434400;
 
-    char *my_string = (char*)"test intersection name";
-    stringstream ss;
-   
-    DescriptiveName_t *update_to_intersection_name = (DescriptiveName_t *)calloc(1, sizeof(DescriptiveName_t));
-    update_to_intersection_name->buf = reinterpret_cast<uint8_t *>(my_string);
-    ss << update_to_intersection_name->buf;
-    ASSERT_EQ(ss.str(), "test intersection name");
-
-    IntersectionReferenceID_t *update_to_intersection_id = (IntersectionReferenceID_t *)calloc(1, sizeof(IntersectionReferenceID_t));
-    update_to_intersection_id->id = 9012;
-
-
-    bool transform_status = ntcip1202_p->ToJ2735r41SPAT(spat_ptr, reinterpret_cast<char *>(update_to_intersection_name->buf), update_to_intersection_id->id);
+    auto ntcip1202_p = std::make_shared<Ntcip1202>();
+    SPAT *spat_ptr = (SPAT *)calloc(1, sizeof(SPAT));
+    ntcip1202_p->ToJ2735SPAT(spat_ptr, tsMsec, "test intersection name", 9012);
     auto _spatMessage = std::make_shared<tmx::messages::SpatMessage>(spat_ptr);
     auto spat = _spatMessage->get_j2735_data();
-    ASSERT_EQ(transform_status, true);
```

Comment:
This assert statement did nothing. Prior to my changes the NTCIP method returned a bool but they returned true regardless of success of failure. I removed this return type since it did not have any meaningful value. That is why I removed the assert statement.